### PR TITLE
Don't hang up tests on SyntaxError

### PR DESCRIPTION
**Pull request by maackle**
- [x] I have added a summary of my changes to the changelog

### CODE REVIEW

**Comment by maackle** [comment]
Yep, this is a real problem. Stay tuned...

File: nodejs_conductor/index.js [hunk 1]
```diff
@@ -72,7 +72,13 @@ Conductor.run = function (config, fn) {
     return new Promise((fulfill, reject) => {
         try {
             conductor._start(callbackFromPromise(fulfill, reject))
-            fn(() => conductor._stop(), conductor)
+            const promise = fn(() => conductor._stop(), conductor)
+            if (promise && promise.catch) {
+                // If the function returned a promise, pass on its potential rejection
+                // to the outer promise
+                // promise.catch(reject)
```
Comment (Connoropolous) [inline nodejs_conductor/index.js]: uncomment?

Comment (maackle) [reply nodejs_conductor/index.js]: omg

Comment (Connoropolous) [reply nodejs_conductor/index.js]: lol

**Pull request merged**